test/remote/gateways/remote_plexo_test.rb

```diff
@@ -31,6 +31,22 @@ def setup
       description: 'Test desc',
       reason: 'requested by client'
     }
+
+    @network_token_credit_card = ActiveMerchant::Billing::NetworkTokenizationCreditCard.new({
+        first_name: 'Santiago', last_name: 'Navatta',
+        brand: 'Visa',
+        payment_cryptogram: 'UnVBR0RlYm42S2UzYWJKeWJBdWQ=',
+        number: '5555555555554444',
+        source: :network_token,
+        month: '12',
+        year: 2020
```

Comment:
❓ Question:

Gateway sandox is not checkking if the NT is expired ?

Reply:
No, it seems the sandbox relies on the number and invoice number combination.